src/util.py

```diff
@@ -297,3 +299,35 @@ def get_clean_rastock_df() -> DataFrame:
     rastock_df = clean_bsb_output_cols(rastock_df)
     rastock_df = convert_column_units(rastock_df)
     return rastock_df
+
+
+# already in dmutils
+def save_figure_to_gcfs(fig, gcspath, figure_format="png", dpi=200, transparent=False):
```

Comment:
Just curious why this is being added back when, as the comment acks, it is in `dmlutils` already. Was this from the time when there were issues with the dependencies in this repo?

Reply:
lol i totally forgot that we had already fixed this issue. ill fix, thanks.